mod_reforged/hooks/config/strings.nut

```diff
@@ -841,7 +848,7 @@ local vanillaDescriptions = [
 				Type = ::UPD.EffectType.Passive,
 				Description = [
 					"Once per battle, upon receiving a killing blow, survive instead with a few hitpoints left and have all damage over time effects (e.g. bleeding, poisoned) cured.",
-					"When this effect triggeres, gain increased stats until your next turn"
+					"When this effect triggers, gain increased stats until your next turn"
```

Comment:
This line should be: `"Upon triggering, gain increased [attributes|Concept.CharacterAttribute] until your next [turn.|Concept.Turn]"`